rewatch/src/cli.rs

```diff
@@ -186,9 +186,6 @@ pub enum Command {
         /// Path to a rescript file (.res or .resi)
         #[command()]
         path: String,
-
-        #[command(flatten)]
-        dev: DevArg,
```

Comment:
we don't have the explicit command to build the dev deps anymore?

Reply:
We do, that is part of BuildArgs.
Here this is about the `compile-args` which the tooling calls to figure out the arguments. Tooling does not know if a file is type:dev or not.

Reply:
👍